diff -BbauNr libsim-2.6.3.orig/base/Makefile.am libsim-2.6.3/base/Makefile.am
--- libsim-2.6.3.orig/base/Makefile.am	2009-02-19 10:34:13.000000000 +0100
+++ libsim-2.6.3/base/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -3,6 +3,8 @@
  file_utilities.f90 geo_coord_class.F90 geo_grid_class.f90 \
  io_units.f90 kinds.F90 missing_values.f90 phys_const.f90 getopt_m.f90 optional_values.f90
 
+libsim_base_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
+
 if SHP
 libsim_base_la_LIBADD = $(SHP_LIBS)
 endif
diff -BbauNr libsim-2.6.3.orig/doc/Makefile.am libsim-2.6.3/doc/Makefile.am
--- libsim-2.6.3.orig/doc/Makefile.am	2008-12-15 11:11:44.000000000 +0100
+++ libsim-2.6.3/doc/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -4,7 +4,7 @@
 all: html/index.html
 
 html/index.html: $(cdoxdeps)
-	doxygen-f $<
+	doxygen $<
 
 
 EXTRA_DIST = Doxyfile.in fpp mainpage.dox breve_presentazione.odp
diff -BbauNr libsim-2.6.3.orig/grib/Makefile.am libsim-2.6.3/grib/Makefile.am
--- libsim-2.6.3.orig/grib/Makefile.am	2008-12-15 11:11:43.000000000 +0100
+++ libsim-2.6.3/grib/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -4,6 +4,8 @@
 AM_FCFLAGS = -I../base
 libsim_grib_la_LIBADD = ../base/libsim_base.la 
 
+libsim_grib_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
+
 if EMOS
 libsim_grib_la_SOURCES += grib_io_class.F90 geo_grid_grib.F90 matchgrib_class.f90
 libsim_grib_la_LIBADD += $(EMOS_LIBS)
diff -BbauNr libsim-2.6.3.orig/log4fortran/Makefile.am libsim-2.6.3/log4fortran/Makefile.am
--- libsim-2.6.3.orig/log4fortran/Makefile.am	2008-12-15 11:11:42.000000000 +0100
+++ libsim-2.6.3/log4fortran/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -8,6 +8,7 @@
 liblog4fortran_la_LIBADD = -llog4c -lcnf
 endif
 
+liblog4fortran_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
 
 nodist_include_HEADERS = log4fortran.mod
 
diff -BbauNr libsim-2.6.3.orig/qc/Makefile.am libsim-2.6.3/qc/Makefile.am
--- libsim-2.6.3.orig/qc/Makefile.am	2008-12-15 11:11:43.000000000 +0100
+++ libsim-2.6.3/qc/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -4,6 +4,8 @@
 AM_FCFLAGS = -I../base -I../vol7d
 libsim_qc_la_LIBADD = ../base/libsim_base.la ../vol7d/libsim_vol7d.la
 
+libsim_qc_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
+
 # c'e` una maniera migliore?
 tmpvar = $(libsim_qc_la_SOURCES:.f90=.mod)
 nodist_include_HEADERS = $(tmpvar:.F90=.mod)
diff -BbauNr libsim-2.6.3.orig/termolib/Makefile.am libsim-2.6.3/termolib/Makefile.am
--- libsim-2.6.3.orig/termolib/Makefile.am	2008-12-15 11:11:48.000000000 +0100
+++ libsim-2.6.3/termolib/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -4,6 +4,8 @@
 AM_FCFLAGS = -I../base
 libsim_termolib_la_LIBADD = ../base/libsim_base.la
 
+libsim_termolib_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
+
 # c'e` una maniera migliore?
 tmpvar = $(libsim_termolib_la_SOURCES:.f90=.mod)
 nodist_include_HEADERS = $(tmpvar:.F90=.mod)
diff -BbauNr libsim-2.6.3.orig/vol7d/Makefile.am libsim-2.6.3/vol7d/Makefile.am
--- libsim-2.6.3.orig/vol7d/Makefile.am	2009-02-19 10:34:14.000000000 +0100
+++ libsim-2.6.3/vol7d/Makefile.am	2009-02-19 10:43:27.000000000 +0100
@@ -15,6 +15,8 @@
 # Librerie aggiuntive
 libsim_vol7d_la_LIBADD = ../base/libsim_base.la
 
+libsim_vol7d_la_LDFLAGS = -version-info @LIBSIM_VERSION_INFO@
+
 if ORSIM
 libsim_vol7d_la_SOURCES += vol7d_oraclesim_class.f90 oraclesim.c
 AM_CPPFLAGS += $(ORA_CPPFLAGS)
